planner/physical.go

```diff
@@ -34,17 +38,58 @@ func (pp *physicalPlanner) Plan(spec *PlanSpec) (*PlanSpec, error) {
 		return nil, err
 	}
 
+	// Convert yields into result list
+	// TODO: Implement this via a transformation rule
+	final, err := removeYields(transformedSpec)
```

Comment:
Curious why you decided to remove yield nodes just before completing physical planning?  Seems like it could be done first thing during logical planning, which might simplify plan representation.

Reply:
Yes you're right I should probably change this. However I do want to remove this fairly quickly and replace it with a rule.